gulpfile.js

```diff
@@ -204,8 +204,13 @@ gulp.task('protractor', function () {
     .pipe(plugins.protractor.protractor({
       configFile: 'protractor.conf.js'
     }))
-    .on('error', function (e) {
-      throw e;
+    .on('end', function() {
+      console.log('E2E Testing complete');
+      process.exit();
```

Comment:
ok, so make this line `process.exit(0);`..should I add a comment above that says `// exit with success.` ?